app/models/miq_server.rb

```diff
@@ -590,11 +590,23 @@ def self.zone_is_modifiable?
     Zone.visible.in_my_region.count > 1
   end
 
+  def self.managed_providers
+    providers = {}
+    ExtManagementSystem.all.each { |ems| providers[ems.type] = ems.class.count }
+
+    providers
+  end
+
   def self.managed_resources
     {
       :vms                     => Vm.active.count,
       :hosts                   => Host.active.count,
       :aggregate_physical_cpus => Host.active.in_my_region.sum(:aggregate_physical_cpus),
+      :providers               => managed_providers,
+      :deployment              => MiqEnvironment::Command.is_appliance? ? "appliance" : "podified",
+      :arch                    => MiqEnvironment::Command.arch.to_s,
+      :services                => Service.count,
+      :service_catalog_items   => ServiceTemplate.count,
```

Comment:
I wonder if we should make a distinction between active services vs retired service vs total services?  For service catalog items, I guess the similar question is whether they are published into a catalog or not.

Reply:
Additionally, I don't think we are using unmanaged_resources method anymore so we can delete that if we aren't.

Reply:
Thoughts on the following?:
```
      :services                => Service.group(:lifecycle_state).count,
      :service_catalog_items   => {:active => ServiceTemplate.active.count, :archived => ServiceTemplate.archived.count},
```

`Service` is a bit weird because it has a `.retired` call but nothing explicit like `.active`

Reply:
🤔 That's weird - wonder if we should add an active method anyway?